RocketChat/Rocket.Chat
# Regression: Fix CORS in uikit endpoints

## Proposed changes (including videos or screenshots)




The issue was introduced by https://github.com/RocketChat/Rocket.Chat/pull/22150 where the validation was wrong when CORS config was disabled.

## Issue(s)


## Steps to test or reproduce

- Make sure CORS is disabled
- Use any UiKit action (like a `/poll` modal creation)
- Without the fix you should get a CORS not allowed error

## Further comments
